docs/pages/deployment/discovery.rst

```diff
@@ -3,27 +3,70 @@
 Discovery
 #########
 
-.. warning::
-    This feature is under development and subject to change.
-
 Discovery allows parties to publish information about themselves as a Verifiable Presentation,
 so that other parties can discover them for further (data) exchange.
 
-In this Discovery Service protocol there are clients and servers: clients register their Verifiable Presentations on a server,
-which can be queried by other clients.
-Where to find the server and what is allowed in the Verifiable Presentations is defined in a Discovery Service Definition.
-These are JSON documents that are loaded by both client and server.
+A Discovery Service is hosted on a server (also a Nuts node), by an organization that is agreed upon by the parties to be the server for that particular use case.
+The parties implementing that use case then configure their Nuts nodes with the service definition that defines the server.
+
+The service definition is a JSON document agreed upon (and loaded) by all parties that specifies:
+
+- which Verifiable Credentials are registered on the service,
+- where the Discovery Service is hosted, and
+- how often the Verifiable Presentations must be updated.
+
+Service definitions are loaded from the ``discovery.definitions.directory`` directory by both client and server.
+It does not load subdirectories. If the directory contains JSON files that are not (valid) service definitions, the node will fail to start.
+
+Clients
+*******
+
+Clients will periodically query the Discovery Service for new registrations.
+Applications can then search for entries in the Discovery Service, e.g.:
+
+.. code-block:: http
+
+    GET /internal/discovery/v1/discovery/addressbook2024:dev/?credentialSubject.name=John%20Doe
```

Comment:
using `addressbook2024:dev` as example might give people the idea to start using `:`. Just use `example`?